### PR TITLE
add new lqm statuses for latency and retries

### PR DESCRIPTION
**Pull request by ab7pa**

### CODE REVIEW

**Review by wu2s — CHANGES_REQUESTED** [review]
Please edit - "latency was too high"

**Comment by ab7pa** [comment]
Which would best express that too much latency is a bad thing?  Any suggestions?  THX
- "latency is too high"
- "latency is too great"
- "latency is too large"

**Comment by wu2s** [comment]
Probably the first alternative.

**Comment by ab7pa** [comment]
Thanks, @wu2s Randy!

File: arednGettingStarted/node_status.rst [hunk 1]
```diff
@@ -119,7 +119,7 @@ Status
   - *blocked - distance*: LQM determined that the remote node is either too close or too distant, based on the Min and Max Distance settings described in the *Advanced Configuration* section.
   - *blocked - signal*: LQM determined that the SNR on the link is too low to reliably pass data, based on the Min SNR setting described in the *Advanced Configuration* section.
   - *blocked - retries*: LQM determined that the retransmission rate was too high to reliably pass data.
-  - *blocked - latency*: LQM determined that the link latency was to long to reliably pass data.
+  - *blocked - latency*: LQM determined that the link latency was to high to reliably pass data.
```
Comment (wu2s) [inline arednGettingStarted/node_status.rst]: too high not to high

**Review by wu2s — CHANGES_REQUESTED** [review]

File: arednGettingStarted/node_status.rst (hunk 1 above)
Comment (ab7pa) [reply arednGettingStarted/node_status.rst]: Yes -- corrected on previous commit.  THX

**Comment by wu2s** [comment]
Approved

**Pull request merged**